stable/efs-provisioner/templates/storageclass.yaml

```diff
@@ -11,8 +11,10 @@ metadata:
 {{- if .Values.efsProvisioner.storageClass.isDefault }}
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
+{{- if .Values.annotations }}
```

Comment:
if i add annotation and not set the `.Values.efsProvisioner.storageClass.isDefault` the annotations will never show up

Reply:
need to do something like:
```
annotations:
{{- if .Values.efsProvisioner.storageClass.isDefault }}
    storageclass.kubernetes.io/is-default-class: "true"
{{- end}}
{{- if .Values.annotations }}
{{ toYaml .Values.annotations | indent 4 }}
{{- end }}
```

Reply:
Yes good point!